2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Configure stats pid to 19244
2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Loading settings from C:\Users\erikw\.config\wandb\settings
2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Loading settings from c:\Users\erikw\Desktop\ExjobbKod\base\wandb\settings
2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-26 14:30:12,922 INFO    MainThread:19244 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-26 14:30:12,923 INFO    MainThread:19244 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\erikw\Desktop\ExjobbKod\base\wandb\run-20240226_143012-a8cqs75n\logs\debug.log
2024-02-26 14:30:12,923 INFO    MainThread:19244 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\erikw\Desktop\ExjobbKod\base\wandb\run-20240226_143012-a8cqs75n\logs\debug-internal.log
2024-02-26 14:30:12,923 INFO    MainThread:19244 [wandb_init.py:init():566] calling init triggers
2024-02-26 14:30:12,923 INFO    MainThread:19244 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'epochs': 10, 'batch_size': 32, 'num_heads': 8, 'num_encoders': 2, 'emb_dim': 128, 'ff_dim': 128, 'lr': 0.0001, 'weight_decay': 0.1, 'vocab_size': 475, 'num_parameters': 1277915}
2024-02-26 14:30:12,923 INFO    MainThread:19244 [wandb_init.py:init():591] re-initializing run, found existing run on stack: ls4kjebm
2024-02-26 14:30:12,925 INFO    MainThread:19244 [wandb_run.py:_finish():1970] finishing run strompfel/TestingMain/ls4kjebm
2024-02-26 14:30:12,926 ERROR   MainThread:19244 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-02-26 14:30:12,926 INFO    MainThread:19244 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-26 14:30:12,926 INFO    MainThread:19244 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-26 14:30:12,926 INFO    MainThread:19244 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-26 14:30:12,926 INFO    MainThread:19244 [wandb_run.py:_restore():2193] restore
2024-02-26 14:30:12,926 INFO    MainThread:19244 [wandb_run.py:_restore():2199] restore done
